ui/backend/src/common/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './types';
 export * from './validation';
 export * from './resources';
 export * from './domains';
+export * from './constants';
```

Comment:
where we're using constants?

Reply:
In the context of this PR, it's about the `ZTPFW_NAMESPACE` constant.
Constants exported at this line are shared between backend and frontend.

Reply:
ok ok